packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -85,6 +155,18 @@ class HorizontalOverflow extends Foundation<HorizontalOverflowProps,  React.AllH
         return super.generateClassNames(get(this.props, "managedClasses.horizontalOverflow"));
     }
 
+    /**
+     * Request's an animation frame if there are currently no open animation frame requests
+     */
+    private requestFrame = (): void => {
+        if (this.openRequestAnimationFrame) {
+            return;
+        }
+
+        this.openRequestAnimationFrame = true;
+        window.requestAnimationFrame(this.onScrollChange);
```

Comment:
You might want to look at https://github.com/wuct/raf-throttle instead of manually setting off the rafThrottle - we can then use that package anywhere we need to use raf instead of implementing that logic in every component.

Reply:
Yep, done.